### ISSUE TITLE
Connection closed: 1006 - connection was closed uncleanly (WebSocket connection upgrade failed (400 - BadRequest))

### ISSUE THREAD

**Issue by ratanlp** [jugaad-py/jugaad-trader#20]
**Describe the bug**
I've started getting this Connection closed: 1006 error for the websocket ticker code.

**Code snippet**
from jugaad_trader import Zerodha

kws = kite.ticker()

def on_ticks(ws, ticks):
    # Callback to receive ticks.
    print("Ticks: {}".format(ticks))

def on_connect(ws, response):
    # Callback on successful connect.
    # Subscribe to a list of instrument_tokens (RELIANCE and ACC here).
    ws.subscribe([738561, 5633])

    # Set RELIANCE to tick in `full` mode.
    ws.set_mode(ws.MODE_FULL, [738561])

def on_close(ws, code, reason):
    # On connection close stop the event loop.
    # Reconnection will not happen after executing `ws.stop()`
    ws.stop()

kws.on_ticks = on_ticks
kws.on_connect = on_connect
kws.on_close = on_close

kws.connect()

**Error**
Connection closed: 1006 - connection was closed uncleanly (WebSocket connection upgrade failed (400 - BadRequest))

**Comment by cmullu** [comment]
Same issue for me. Please resolve ASAP

**Comment by sevakram** [comment]
I have reproduced the issue, It will take some time, may be till weekend.

It will greatly help if you can debug and find the root cause.

**Comment by ratanlp** [comment]
This is the exception I see:

```
File "/opt/anaconda3/lib/python3.8/site-packages/kiteconnect/ticker.py", line 569, in subscribe
    self.ws.sendMessage(
AttributeError: 'NoneType' object has no attribute 'sendMessage'

```

**Comment by sevakram** [comment]
Resolved by PR #23 , Thanks @iamknownstranger
